Github-case-study.md

```diff
@@ -0,0 +1,183 @@
+---
+title: GitHub Case Study
+keywords: development platforms, GitHub, 
+tags: Development Platforms, GitHub, Category Name
+sidebar: development-platforms
+permalink: development-platforms/case-study-GitHub.html
+folder: development-platforms
+---
+# <p align="center"><span>GitHub Case Study<span></p>
+
+<p align="center"><img src="Images/github-mark.png"><p>
+
+## Table of contents
+
+1. [Introduction](#introduction)
+2. [Brief History](#brief-history)
+3. [Features](#features)
+4. [Strengths](#strengths)
+5. [Weaknesses](#weaknesses)
+6. [Comparison](#comparison)
+7. [Summary](#summary)
+8. [Credits](#credits)
+9. [References](#references)
+
+
+## Introduction
+
+Effective teamwork and code management are essential for the development of software. Finding the ideal platform to assist this procedure is the challenge. GitHub emerged as a solution for these challenges by providing a web-based platform for code storage, sharing, and collaboration.
+
+With the help of a collection of tools for collaboration and code editing, GitHub seeks to give developers a single location to save, organize, and manage their code. It also contains tools for tracking bugs and feature requests, such as issues and pull requests for reviewing and merging code changes. Continuous integration and deployment are two additional technologies that GitHub supports and integrates. For example, Github has released desktop apps for Windows and macOS that make it easy for users to integrate it into their workflow. GitHub provides a full and easy solution for software development. 
+
+  
+
+ ## Brief History
+
+GitHub's development started on October 19, 2007. In April 2008, Tom Preston-Werner, Chris Wanstrath, P.J. Hyett, and Scott Chacon officially launched the site after beta testing.Since its founding in 2008, GitHub has grown to become one of the most popular services for hosting and exchanging code.  
```

Comment:
```suggestion
GitHub's development started on October 19, 2007. In April 2008, Tom Preston-Werner, Chris Wanstrath, P.J. Hyett, and Scott Chacon officially launched the site after beta testing. Since its founding in 2008, GitHub has grown to become one of the most popular services for hosting and exchanging code.  
```